pom.xml

```diff
@@ -112,6 +139,38 @@
 
 	<build>
 		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
```

Comment:
Do we need 2 same plugins in one POM.xml? `<artifactId>maven-surefire-plugin</artifactId>` line 144 and line 226

Reply:
Nope, going to remove it, thanks